apps/desktop-wallet/src/components/AddressColorIndicator.tsx

```diff
@@ -38,7 +38,7 @@ const AddressColorIndicator = ({
   className
 }: AddressColorIndicatorProps) => {
   const address = useAppSelector((s) => selectAddressByHash(s, addressHash))
-  const isPassphraseUsed = useAppSelector((s) => s.activeWallet.passphrase)
+  const isPassphraseUsed = useAppSelector((s) => s.activeWallet.isPassphraseUsed)
```

Comment:
Since we are not storing the passphrase anymore, I replaced it with a boolean flag.